.github/dependabot.yml

```
@@ -1,40 +1,40 @@
+      day: "monday"
+      time: "02:00"
+      timezone: "America/New_York"
+    open-pull-requests-limit: 10
+    labels:
+      - "dependencies"
+    groups:
+      minor-and-patch:
+        update-types:
+          - "minor"
+          - "patch"
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (1 > 0)

(empty-lines)